### DISCUSSION TITLE
umi4 使用mfsu启动报错

### DISCUSSION THREAD

**Discussion by weeeeibin** [umijs/umi#8491] [General]

**Comment by weeeeibin** [comment]
请问大佬这是为什么呢 Monorepo项目 关闭mfsu启动正常

**Comment by fz6m** [comment]
给一个复现 demo 看看。

**Reply by weeeeibin** [reply]
在demo里没办法复现项目中的问题！
我修改了 node_modules/.cache/mfsu/mf-va_umi-request.js 打印出了这个文件返回的undefined
![image](https://user-images.githubusercontent.com/23420710/179698873-739eb5ca-3734-46fc-9b83-b8535be887f1.png)
